lemminx-maven/src/test/java/org/eclipse/lemminx/maven/test/LocalPluginTest.java

```diff
@@ -46,65 +48,105 @@ public void tearDown() throws InterruptedException, ExecutionException {
+	public void testPluginArtifactHover()
+			throws IOException, InterruptedException, ExecutionException, URISyntaxException, TimeoutException {
+		assertTrue(languageService
+				.doHover(createDOMDocument("/pom-plugin-artifact-hover.xml", languageService), new Position(14, 18),
+						new XMLHoverSettings())
+				.getContents().getRight().getValue().contains("Maven Surefire MOJO in maven-surefire-plugin"));
 	}
-	
+
 	// Diagnostic related tests
 
-		@Test(timeout=30000)
-	 	public void testPluginConfigurationDiagnostics() throws IOException, InterruptedException, ExecutionException, URISyntaxException {
-			DOMDocument document = createDOMDocument("/pom-plugin-configuration-diagnostic.xml", languageService);
-			assertTrue(languageService.doDiagnostics(document, () -> {}, new XMLValidationSettings()).stream().map(Diagnostic::getMessage)
-					.anyMatch(message -> message.contains("Invalid plugin configuration")));
-			assertTrue(languageService.doDiagnostics(document, () -> {}, new XMLValidationSettings()).size() == 2);
-		}
-		
-		@Test
-	 	public void testPluginGoalDiagnostics() throws IOException, InterruptedException, ExecutionException, URISyntaxException {
-			DOMDocument document = createDOMDocument("/pom-plugin-goal-diagnostic.xml", languageService);
-			assertTrue(languageService.doDiagnostics(document, () -> {}, new XMLValidationSettings()).stream().map(Diagnostic::getMessage)
-					.anyMatch(message -> message.contains("Invalid goal for this plugin")));
-			assertTrue(languageService.doDiagnostics(document, () -> {}, new XMLValidationSettings()).size() == 2);
-		}
+	@Test(timeout = 30000)
+	public void testPluginConfigurationDiagnostics()
+			throws IOException, InterruptedException, ExecutionException, URISyntaxException {
+		DOMDocument document = createDOMDocument("/pom-plugin-configuration-diagnostic.xml", languageService);
+		assertTrue(languageService.doDiagnostics(document, () -> {
+		}, new XMLValidationSettings()).stream().map(Diagnostic::getMessage)
+				.anyMatch(message -> message.contains("Invalid plugin configuration")));
+		assertTrue(languageService.doDiagnostics(document, () -> {
+		}, new XMLValidationSettings()).size() == 2);
+	}
+
+	@Test
+	public void testPluginGoalDiagnostics()
+			throws IOException, InterruptedException, ExecutionException, URISyntaxException {
+		DOMDocument document = createDOMDocument("/pom-plugin-goal-diagnostic.xml", languageService);
+		assertTrue(languageService.doDiagnostics(document, () -> {
+		}, new XMLValidationSettings()).stream().map(Diagnostic::getMessage)
+				.anyMatch(message -> message.contains("Invalid goal for this plugin")));
+		List<Diagnostic> diags = languageService.doDiagnostics(document, () -> {
+		}, new XMLValidationSettings());
+		System.out.println(diags);
+		assertTrue(languageService.doDiagnostics(document, () -> {
+		}, new XMLValidationSettings()).size() == 2);
+	}
+
+	// Tests regressions related to
```

Comment:
This file was accidentally formatted... the real change to LocalPluginTest is here.